src/components/StatsGathering/ForceBrowserShareAssets.tsx

```diff
@@ -2,12 +2,14 @@
 /* eslint-disable jsx-a11y/alt-text */
 interface Props {
   visibleForces: string
+  userId: number | undefined
 }
 
-const ForceBrowserShareAssets = ({ visibleForces }: Props) => {
+const ForceBrowserShareAssets = ({ visibleForces, userId }: Props) => {
   const forces = visibleForces.split(",")
+  const id = userId?.toString()
   const forcesImgs = forces.map((force: string, i: number) => (
-    <img src={`/forces.png?forceID=${force}`} className="govuk-!-display-none" key={i} />
+    <img src={`/forces.png?forceID=${force}?userID${id}`} className="govuk-!-display-none" key={i} />
```

Comment:
This needs a tweak - you've got two `?` and no `=`

Reply:
Thanks, updated to the below. 
```
<img src={`/forces.png?forceID=${force}&userID=${id}`} className="govuk-!-display-none" key={i} />
```